src/app/libp2p_helper/src/libp2p_helper/main.go

```diff
@@ -1122,7 +1115,6 @@ func main() {
 	logging.SetLogLevel("autorelay", "info") // Logs relayed byte counts spammily
 	logging.SetLogLevel("providers", "debug")
 	logging.SetLogLevel("dht/RtRefreshManager", "warn") // Ping logs are spammy at debug, cpl logs are spammy at info
-	logging.SetLogLevel("dht", "info") // Logs every operation to debug
```

Comment:
Can you re-add this? This sets the minimum level, so deleting it shows *all* logs from the subsystem.